pillarbox-core-business/src/main/java/ch/srgssr/pillarbox/core/business/SRGMediaItem.kt

```diff
@@ -130,28 +143,33 @@ class SRGMediaItemBuilder internal constructor(mediaItem: MediaItem) {
     }
 
     /**
-     * Set force location
+     * Forces the location for IL/SAM backend calls.
      *
-     * @param forceLocation The location to use on the IL/SAM backend calls. Can be `null`, `CH`,  or `WW`.
+     * @param forceLocation The location to force. Valid values are:
+     *  - `null`: disables forced location and uses automatic detection.
+     *  - `"CH"`: forces the location to Switzerland.
+     *  - `"WW"`: forces the location to Worldwide.
      */
     fun forceLocation(forceLocation: String?) {
```

Comment:
Since the list of valid values is finite wouldn't it be better to introduce an `enum` here? This would avoid the need to document valid values extensively.

Reply:
Indeed. In this PR, I only wanted to focus the changes on the documentation itself.
I've created a draft issue to introduce an enum for the location. I'll complete it later so we can work on it.